web/src/components/tasks/anytime-action/tax-clearance-certificate/AnytimeActionTaxClearanceCertificateElementAlert.tsx

```diff
@@ -0,0 +1,71 @@
+import { Alert } from "@/components/njwds-extended/Alert";
+import { useConfig } from "@/lib/data-hooks/useConfig";
+import { getProfileConfig } from "@/lib/domain-logic/getProfileConfig";
+import { ProfileContentField } from "@/lib/types/types";
+import { ReactElement } from "react";
+
+interface Props {
+  fieldErrors: string[];
+}
+
+export const AnytimeActionTaxClearanceCertificateElementAlert = (props: Props): ReactElement | null => {
+  const { Config } = useConfig();
+
+  const displayTaxClearanceErrorAlert = (): boolean => props.fieldErrors.length > 0;
```

Comment:
we can remove the fn - `const displayTaxClearanceErrorAlert = props.fieldErrors.length > 0;`